doc/user/using-java9.md

```diff
@@ -30,8 +31,19 @@ As another check you can run a loop and see that it is compiled.
 $ bin/truffleruby \
     -J-XX:+UnlockExperimentalVMOptions \
     -J-XX:+EnableJVMCI \
-    -J--module-path=$GRAAL_REPO/sdk/mxbuild/modules/org.graalvm.graal_sdk.jar:$GRAAL_REPO/truffle/mxbuild/modules/com.oracle.truffle.truffle_api.jar:$GRAAL_REPO/compiler/mxbuild/modules/jdk.internal.vm.compiler.jar \
-    -J-Dgraal.TraceTruffleCompilation=true \
+    -J--module-path=$GRAAL_REPO/sdk/mxbuild/modules/org.graalvm.graal_sdk.jar:$GRAAL_REPO/truffle/mxbuild/modules/com.oracle.truffle.truffle_api.jar \
+    -J--upgrade-module-path=$GRAAL_REPO/compiler/mxbuild/modules/jdk.internal.vm.compiler.jar \
```

Comment:
The line
`-J-Dgraal.TraceTruffleCompilation=true \`
was inadvertently removed.